scripts/firmware.py

```diff
@@ -0,0 +1,155 @@
+import os
+
+CONFIG_DIR: str = os.path.expanduser("~/printer_data/config")
+KLIPPER_DIR: str = os.path.expanduser("~/klipper")
+KATAPULT_DIR: str = os.path.expanduser("~/katapult")
+
+DEBUG = True
+FLASHER_VERSION: str = "0.0.1"
+
+
+class Color(str):
+    RESET = "\033[0m"
+    RED = "\033[91m"
+    GREEN = "\033[92m"
+    YELLOW = "\033[93m"
+    MAGENTA = "\033[95m"
+    CYAN = "\033[96m"
+
+    def colored_text(self, text: str, color: str) -> str:
+        return f"{color}{text}{self.RESET}"
+
+
+class Menu:
+    def __init__(self, title: str, menu_items: dict):

```

Comment:
```suggestion
    def __init__(self, title: str, menu_items: dict: [str, MenuItem]):
```
Make a MenuItem class? 😄

Reply:
Hows it look now?